docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,442 @@
+  This information identifies the node with the freshest cluster state, which
+  minimizes the quantity of data that might be lost.
 
-* Run the ``crate-node detach-cluster`` tool, described below, on every other
-  node in the cluster.
+Example
+~~~~~~~
 
-* Start all other nodes and verify that each one joins the cluster.
+If the first node reports (4, 12) and a second node reports (5, 3), then prefer
+the second node for its larger term. If the second node reports (3, 17), then
+you prefer the first node for its larger term. If the second node reports
+(4, 10) then it has the same term as the first node but a smaller version, so
+then you prefer the first node.
 
-* Investigate the data in the cluster to discover if any was lost during this
-  process.
+You should only run this tool if you have irrecoverably lost half or more
+of the master-eligible nodes in this cluster. Also only run this tool if you
+cannot restore the cluster from a snapshot. If you have multiple surviving
+master-eligible nodes, you should run this tool on the node with the highest
+cluster state (term, version) pair.
 
-* When you run the tool it will make sure that the node that is being used
-  to bootstrap the cluster is not running. It is important that all other
-  master-eligible nodes are also stopped while this tool is running, but
-  the tool does not check this.
+Suppose your cluster has had five master-eligible nodes and you have
+irrecoverably lost three of them, leaving two nodes remaining. In such a case
+you can unsafely bootstrap the cluster.
 
-The message `Master node was successfully bootstrapped` does not mean that
-there has been no data loss, it just means that tool was able to complete its job.
+.. NOTE::
 
-Detach a node from a cluster
-----------------------------
+    Before you run ``crate-node unsafe-bootstrap`` you need to stop CrateDB.
 
-When a node joins a cluster initially, the unique cluster id will be stored in
-its nodes metadata. This prevents the node from joining a cluster with a different
-cluster id. There are situations where it necessary to reset the cluster id from
-a node. When a cluster is not recoverable, it might be worth moving nodes
-to a new cluster. This is in particular useful when the voting configuration
-of a node is reset, and a new cluster is bootstrapped.
+.. WARNING::
 
-It is unsafe for nodes to move between clusters, because different clusters have
-completely different cluster metadata. There is no way to safely merge the
-metadata from two clusters together.
+    This tool can cause arbitrary data loss, and its use should be your last
+    resort.
 
-To protect against inadvertently joining the wrong cluster, each cluster creates
-a unique identifier, known as the cluster UUID, when it first starts up. Every
-node records the UUID of its cluster and refuses to join a cluster with a
-different UUID.
+* Run ``crate-node unsafe-bootstrap`` on the first remaining node
```

Comment:
rephrase these sentences so that you're not duplicating the command in the prose. just intro the command with a description of what you're doing